ikarus/finiteelements/mechanics/materials/neohooke.hh

```diff
@@ -71,7 +77,7 @@ struct NeoHookeT : public Material<NeoHookeT<ST>>
     if constexpr (!Concepts::EigenVector<Derived>) {
       const auto traceC  = C.trace();
       const auto logdetF = log(sqrt(C.determinant()));
-      return lambdaAndmu_.mu / 2.0 * (traceC - 3 - 2 * logdetF) + lambdaAndmu_.lambda / 2.0 * logdetF * logdetF;
```

Comment:
Why did you add the brackets here? Just out of curiostity.

Reply:
Good question ... 
I think i might have added an `eval()` when i traced the other bug